fgd/brush/trigger/trigger_momentum_timer_stage_end.fgd

```diff
@@ -0,0 +1,6 @@
+@SolidClass appliesto(MOMENTUM)
+= trigger_momentum_timer_stage_end : "Zone that stops the stage timer."
```

Comment:
```suggestion
= trigger_momentum_timer_stage_end : "Zone that finishes the timer for a stage track. If not used for a stage, the start of the following stage (or map end) will automatically become the stage end. This zone is not used by the full main map track."
```

Reply:
Actually now that I think about it, this hammer entity should probably also generate a checkpoint for the stage of the main track (so there would be one volume that is used as the last checkpoint of the stage in the main track, and also used as the stage end zone for the stage track). Except if the mapper wants the stage to have non-required checkpoints, that might be a problem... The fully robust solution would be to have a *separate* flag for "the final checkpoint of the stage is required" ...but that might be overkill and we can always add it later.

So I guess ignore the part about "not used by the full main map track" in my suggested edit.